documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -2200,6 +2201,66 @@ Contains a structure with two fields:
+{prodname} then uses these JDBC column types to define the schema for the snapshot events.
+After the connector transitions from the snapshot to the streaming phase, the change event schema that results from the default mapping can lead to inconsistent mappings for `BOOLEAN` columns.
+To help ensure that MySQL emits `BOOLEAN` columns uniformly, you can apply the custom  `TinyIntOneToBooleanConverter`, as shown in the following configuration example.
+
+.Example: `TinyIntOneToBooleanConverter` configuration
+[source]
+----
+converters=tinyint-one-to-boolean
+converters.tinyint-one-to-boolean.type=io.debezium.connector.mysql.converters.TinyIntOneToBooleanConverter
+converters.tinyint-one-to-boolean.selector=.*.MY_TABLE.DATA
+converters.tinyint-one-to-boolean.length.checker=false
+----
+
+In the preceding example, the `selector` and `length.checker` properties are optional.
+When specifying the `length.checker` as `false`, the converter will not explicitly confirm that the `TINYINT` data type conforms to a length of `1`.
+By default, the converter checks that `TINYINT` data types conform to a length of `1`.
+Set `length.checker` to `false` to prevent the converter from explicitly confirming the length.
+The `selector` designates the tables or columns to convert, based on the supplied regular expression.
+If you omit the `selector` property, the converter maps all `TINYINT` columns to logical `BOOL` field types.
+To map `TINYINT` columns to `TINYINT(1)` when the `selector` option is not present , omit the `length.checker` property, or set its value to `true`.
+
+=== JDBC sink data types
+
+If you integrate the {prodname} JDBC sink connector with a {prodname} MySQL source connector, the MYSQL connector emits some column attributes differently during the snapshot and streaming phases.
+For the JDBC sink connector to consistently consume changes from both the snapshot and streaming phase, you must include the `JdbcSinkDataTypesConverter` converter as part of the MySQL source connector configuration, as shown in the following example:
+
+.Example: `JdbcSinkDataTypesConverter` configuration
+[source]
+----
+converters=jdbc-sink
+converters.jdbc-sink.type=io.debezium.connector.mysql.converters.JdbcSinkDataTypesConverter
+converters.jdbc-sink.selector.boolean=.*.MY_TABLE.BOOL_COL
+converters.jdbc-sink.selector.real=.*.MY_TABLE.REAL_COL
+converters.jdbc-sink.selector.string=.*.MY_TABLE.STRING_COL
+converters.jdbc-sink.treat.real.as.double=true
+----
+
+In the preceding example, the `selector.*` and `treat.real.as.double` configuration properties are optional.
+
+The `selector.*` properties specify comma-separated lists of regular expressions that specify which tables and columns that the converter applies to.
+If you omit a specific selector property for a field type, the converter processes all fields that match that rule.
```

Comment:
```suggestion
If you do not specify a `selector` property for a field type, when {prodname} emits an event, the converter uses the rule that you specify in the configuration to map the data types of all columns of that type.
```
Taking another stab at this, bc, reading through L2254 again, I realize that it's not quite clear. I don't think that the original text, is any more clear: _"If a specific selector property is omitted for a given field type, then all fields matching that rule will be converted"_

So, if I don't set a `selector` for columns of a specific data type, the existing sentence states that all fields that match some `rule` are converted. 
I'd like to explicitly call out what `rule` we're referring to:

- The default rules for `BOOLEAN` and `REAL` data types that we mention in the paragraphs that follow. 
- The rule defined by the `treat.real.as.double` property.

Based on the current order in which we present the information, it's ambiguous as to whether `rule` might refer to the regex defined for the selector, -- which, of course makes `0` sense in the context of the case where no selector is set.  

I wonder whether we should start this section with a description of the main/default function of the converter (the information in L2255-2262). Or at least explicitly reference the "default converter rules" or the specific converter function that you define in the configuration (is this limited to `treat.real.as.double` or are there others?).

Reply:
So these `selectors` we we refer to have a default rule that accepts all combinations.  So, in this case, if the user does not provide a `selector.boolean` configuration option, by default, we match all boolean data types.  So the net effect is that the user can reduce the scope of the selector by providing their own rule if that's preferred.

Reply:
```suggestion
By default, the converter applies the following rules apply to all Boolean, real, and string-based column data types, across all tables:

* `BOOLEAN` data types are always emitted as `INT16` logical types, with `1` representing `true` and `0` representing `false`
* `REAL` data types are always emitted as `FLOAT64` logical types.
* String-based columns always include the `__debezium.source.column.character_set` schema parameter that contains the column's character set.

For each data type, you can configure a selector rule to override the default scope and apply the selector to specific tables and columns only.
For example, to set the scope of the Boolean converter, add the following rule to the connector configuration, as in the  preceding example: `converters.jdbc-sink.selector.boolean=.*.MY_TABLE.BOOL_COL`
```

OK. I think that we can do something like this ^^ 

I wanted to leave the existing conversation comments intact, so I didn't replace those, but maybe we can use this in place of L2254-2263?